src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,105 @@
 package core.basesyntax;
 
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final int ENSURE_CAPACITY_DIVIDER = 2;
```

Comment:
```suggestion
    private static final double CAPACITY_INDEX = 1.5;
```
ArrayList grows to 1.5